### ISSUE TITLE
[HOLD #40508] [$250]  iOS - Chat - Contacts in the mention suggestion list overlap

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#40590]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.63-0
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:** N/A
**Issue reported by:** Applause - Internal Team

## Action Performed:
1. Launch New Expensify app
2. Go to chat
3. Type @
4. Select a mention from the suggestion list
5. Type @

## Expected Result:
The content in the list will not overlap

## Actual Result:
The content in the list overlaps

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [ ] Android: Native
- [ ] Android: mWeb Chrome
- [x] iOS: Native
- [ ] iOS: mWeb Safari
- [x] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/555d8876-159f-46e3-b952-11ccf07fdf92

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~01754fc1f82dc0f2e2</li>
        <li>Upwork Job ID: 1782542904959717376</li>
        <li>Last Price Increase: 2024-04-29</li>
    </ul>
</details>

**Comment by lanitochka17** [comment]
@MitchExpensify  FYI I haven't added the External label as I wasn't 100% sure about this issue. Please take a look and add the label if you agree it's a bug and can be handled by external contributors

**Comment by lanitochka17** [comment]
We think that this bug might be related to #vip-vsp

**Comment by MitchExpensify** [comment]
Reproduced, I don't see another report of this, labelling external

**Comment by MitchExpensify** [comment]
Waiting on proposals, reassigning as I'm going OOO until the 7th; I'll take it back upon return! 🙇

**Comment by MitchExpensify** [comment]
Not overdue, waiting on proposals

**Comment by Pujan92** [comment]
I think this is related to the new-arch implementation of FlashList, one more issue is raised [here](https://github.com/Expensify/App/issues/40508) for iOS mention selection.

@WoLewicki Do you think swapping with FlatList will be fine here? [ref](https://github.com/Expensify/App/pull/40383)

**Comment by ikevin127** [comment]
- https://github.com/Expensify/App/issues/40508

Above issue already has a [proposal](https://github.com/Expensify/App/issues/40508#issuecomment-2083935664) awaiting review, which could potentially fix both the overlap (this issue) and the mention issue (see above). The proposal also mentions swapping `FlashList` with `FlatList` as alternative solution.

I think this one can be put on hold for the other issue since that already had a bounty increase.

**Comment by Pujan92** [comment]
Agree with @ikevin127.

**Comment by mvtglobally** [comment]
Issue not reproducible during KI retests. (First week)

**Comment by MitchExpensify** [comment]
On hold for https://github.com/Expensify/App/issues/40508

**Comment by johncschuster** [comment]
Thanks for the comment, @MitchExpensify! I'll watch for #40508 to close and then see if this behavior persists.

**Comment by MitchExpensify** [comment]
Back from OOO so taking this back @johncschuster

**Comment by WoLewicki** [comment]
> @WoLewicki Do you think swapping with FlatList will be fine here?

Sorry for the late response. I think we can try and go with such option but is it not the case that there can be many many items in such list based on how many users can be listed there? I think it would be best to try and investigate the root of the problem, but since it can take much time, we could try to swap it with `FlatList` as long as it works correctly 🚀

**Comment by MitchExpensify** [comment]
On hold, Melvin

**Comment by MitchExpensify** [comment]
https://expensify.slack.com/archives/C02NK2DQWUX/p1716245342602619

**Comment by MitchExpensify** [comment]
On hold

**Comment by MitchExpensify** [comment]
On hold for for https://github.com/Expensify/App/issues/40508

**Comment by Pujan92** [comment]
I am not able to reproduce this issue as other [PR](https://github.com/Expensify/App/issues/40508) of using Flatlist seems solved this issue. 
@lanitochka17 Could you plz confirm

**Comment by Pujan92** [comment]
Bump @lanitochka17 for the retest.

**Comment by lanitochka17** [comment]
Issue is not reproducible



https://github.com/Expensify/App/assets/78819774/945ad1f1-4b8f-4dd8-a66d-6fad72df34c9

**Comment by Pujan92** [comment]
Thanks @lanitochka17 , @MitchExpensify We can close this issue.
